docs/content/documentation/systems/Materials/phase_field/MixedSwitchingFunctionMaterial.md

```diff
@@ -0,0 +1,17 @@
+
+# MixedSwitchingFunctionMaterial
+!description /Materials/MixedSwitchingFunctionMaterial
+
+$$
+k(3\eta^2-2\eta^3)+(1-k)(4\eta^3-3\eta^4)
+$$
+
+$$
+k(2\eta^2-\eta^4)+(1-k)(3\eta^4-2\eta^6)
+$$
+
```

Comment:
please at least mention MIX234 and MIX246 here. Just two terms is a bit weak for documentation.